assets/prototype/domain/eventEditor/data/initialization/test/useCacheRehydration.test.ts

```diff
@@ -0,0 +1,71 @@
+import { renderHook } from '@testing-library/react-hooks';
+
+import useCacheRehydration from '../useCacheRehydration';
+import useDatetimes from '../../queries/datetimes/useDatetimes';
+import useTickets from '../../queries/tickets/useTickets';
+import usePriceTypes from '../../queries/priceTypes/usePriceTypes';
+import { ApolloMockedProvider } from '../../../context';
+
+describe('useCacheRehydration', () => {
+	it('checks for datetimes rehydration', async () => {
+		const {
+			result: { current: datetimesList },
+		} = renderHook(
+			() => {
+				useCacheRehydration();
+				return useDatetimes();
+			},
+			{
+				wrapper: ApolloMockedProvider(),
+			}
+		);
+
+		expect(datetimesList.length).toBeGreaterThan(1);
+
+		expect(datetimesList[0]).toHaveProperty('id');
+
+		expect(datetimesList[1]).toHaveProperty('capacity');
```

Comment:
So `datetimesList` will always be an array with more than 2 items ?

Reply:
I have added two items to the test data 😄